libs/ui-watt/src/lib/components/tabs/tabs.component.ts

```diff
@@ -39,4 +39,17 @@ export class WattTabsComponent {
    */
   @ContentChildren(WattTabComponent)
   public readonly tabElements: QueryList<WattTabComponent> = new QueryList<WattTabComponent>();
+  activeTabIndex = 0;
+
+  emitSelectedTabChange(selectedIndex: number) {
+    this.activeTabIndex = selectedIndex;
+    const currentTab = this.tabElements.find(
+      (tab, index) => index === selectedIndex
+    );
+    currentTab?.emitChange();
+  }
+
+  triggerChange() {
```

Comment:
Question: where is `triggerChange()` used or going to be used? :)

Reply:
In our case it is going to be used when we open a drawer with tabs inside. Then we would like to make the active tab emit an event, so we can get the correct data inside the tab content. 

I hope it makes sense :)